cmd/zq/zq.go

```diff
@@ -247,7 +247,7 @@ func (c *Command) Run(args []string) error {
 	if len(readers) == 1 {
 		reader = readers[0]
 	} else {
-		reader = scanner.NewCombiner(readers)
+		reader = scanner.NewCombiner(readers, paths)
```

Comment:
I can't comment on `loadInputs` so I'll comment here. If you load files with `scanner.OpenFile` you don't need to do this- expect for the case of stdin, in which case I recommend wrapping with the `namedReader` type I referenced below.

Reply:
I originally had to change `loadInputs` due to the NewCombiner interface change, but that's no longer needed with your suggestion to pass a Stringer.